src/extension.ts

```diff
@@ -79,16 +79,18 @@ const openSettingsCommand = "workbench.action.openSettings";
 let treeViews: MetalsTreeViews | undefined;
 let currentClient: LanguageClient | undefined;
 
-let worksheetDecorationType: TextEditorDecorationType =
-  window.createTextEditorDecorationType({
+let worksheetDecorationType: TextEditorDecorationType = window.createTextEditorDecorationType(
```

Comment:
You should be able to run `yarn format ` to fix it (maybe also `yarn install` before to update the prettifier)